@xen-orchestra/lite/src/components/AppLogin.vue

```diff
@@ -42,12 +52,17 @@ const password = ref("");
 const error = ref<string>();
 const passwordRef = ref<InstanceType<typeof FormInput>>();
 const isInvalidPassword = ref(false);
+const rememberMe = useLocalStorage("rememberMe", false);
 
 const focusPasswordInput = () => passwordRef.value?.focus();
+const onChangeRememberMe = (value: boolean) => (rememberMe.value = value);
 
 onMounted(() => {
-  xenApiStore.reconnect();
-  focusPasswordInput();
+  if (rememberMe.value) {
+    xenApiStore.reconnect();
```

Comment:
The problem is that the XenApiStore stores the current session ID in local storage, whether this box is checked or not.

I think this is a major security issue because even if someone doesn't check the "Remember me" checkbox, the session ID is still stored and locally accessible, even after the browser is closed.